core/src/main/scala/kafka/log/Log.scala

```diff
@@ -2670,6 +2645,34 @@ object Log {
   private def isLogFile(file: File): Boolean =
     file.getPath.endsWith(LogFileSuffix)
 
+  private def loadProducersFromLog(producerStateManager: ProducerStateManager, records: Records): Unit = {
```

Comment:
Perhaps this could be name loadProducersFromRecords?

Reply:
Done